### ISSUE TITLE
"Ignoring surfaces... the max is 32 surfaces" warning (resulting in missing texture) on nightly builds binaries

### ISSUE THREAD

**Issue by The-Gig** [OpenArena/engine#58]
An user on Discord reported to me about a problem he noticed while using a certain third party model with OA nightly build binaries.

The model is this one
http://en.ws.q3df.org/models/downloads/q3mdl-q4hybrid.zip
it's a quite "heavy" model, the zip even contains two pk3 files, one with lower and one with higher polycount. 

If using nightly builds binaries, the model's face is shown as "missing texture", as are the "holograms" it generates during its idle animation. In console there is "WARNING: Ignoring surfaces in 'models/players/q4hybrid/lower_classica.skin', the max is 32 surfaces!" message.

If using classic OA 0.8.8 binaries, instead, that specific warning is not shown and model's face and its "holograms" are shown correctly.

Here you can see a comparison between classic 0.8.8 binaries and nightly build binaries:
![shot0093](https://user-images.githubusercontent.com/23258514/58465266-9d138f80-8137-11e9-9d4a-49397bf22873.jpg)
![shot0094](https://user-images.githubusercontent.com/23258514/58465267-9dac2600-8137-11e9-9354-292fd619eeb1.jpg)
(By the way, by doing this test I see that playing the game with nightly build binaries -with same settings- it looks noticeably _darker_).

Side notes: using the model shows some issues in general, I just mention them here to say it's not a model I think I would actually use, despite some cool things:
- I don't like the fact that in the idle animation the model moves away from its hitbox, making it an harder target.
- Some of its skins appear to do rely in some Q3A textures, so with some skins part of its body is just transparent. This seems to happen the same with 0.8.8 and nightly builds binaries.

**Comment by leilei-** [comment]
Welcome to 'art piece models'.  The Q4* models were originally intentionally pushing the limits of Q3 in unsafe ways to support way too many texture usage and way too many vertex morphed frames.

**Comment by leilei-** [comment]
Also it "looks darker" because you were using intensity  (aka cheater's brightness) in your first shot.  Overbrights and gamma should properly work in the latest.

**Comment by KuehnhammerTobias** [comment]
https://github.com/ioquake/ioq3/issues/261

**Comment by The-Gig** [comment]
So, there is already a fix which might be backported? https://github.com/ioquake/ioq3/commit/904bbc1a8fd83d6798c35975726440e4b8221d31
Thank you Kuehnhammer...

**Comment by KuehnhammerTobias** [comment]
Yeah, with this fix the model loads without any issues again. I mean, I don't know OpenArena code by heart, but because it's basically ioquake3 I think the fix should work for OpenArena too.
